_posts/2020-07-15-from-engineering-to-product-management-and-back-again.md

```diff
@@ -0,0 +1,261 @@
+right amount of concise... and, as a fairly verbal person how to draw good diagrams and pictures to communicate my
+ideas. There's no shortcut to improving these skills. Getting better is a matter of practice and feedback.
+
+## Learning to say no
+
+One of the most difficult things for me to learn was how and when to say "no." As an engineer, I prided myself on
+having a can-do attitude and making seemingly impossible things happen. As a product manager, though, I have to
+make sure that we're always building the right thing for our users. Since my job is to make sure we're doing that,
+I usually have conviction that we're currently doing the right thing. That means that, on average, when someone
+requests product resources or proposes a new feature, I have to say no.
+
+I like to be helpful, and so I was terrible at saying no at first. I wanted to help anyone and everyone who came to
+me because, like when I was an engineer, I wanted solve everyone's problems. But as I focused more and more on user
+needs, it became clear what we should be working on to have the best impact on our users. That gave me the
+conviction I needed to start saying no.
+
+That still didn't make it easier to _say_ no. Prioritization decisions can have a big impact on other teams, and
+it's hard to see your coworkers feel disappointed when something they want won't be prioritized. Learning how to
+say no kindly and empathetically is an art form. But in general, it's best done by clearly communicating priorities
+and correctly setting expectations in the first place so that people don't come with requests that won't get
+fulfilled. Doing that involves educating your coworkers on how the product prioritization process works,
+socializing your product roadmap and spending lots of time to answer questions. It also may involve looping
+stakeholders into the prioritization process so that their voices are clearly heard and they feel ownership over
+the roadmap. It's much easier to get things done when everyone is in alignment from the beginning.
+
+# Coming back to engineering
+
+After the first six months of working as a product manager I felt like I had gained my "product legs." I had
+launched some successful features, had weathered major organizational change and had a clear idea of what our users
+needed and what the future ought to look like. Over the next year I continued to hone my skills and expand my scope
+as a product manager. Overall, things were going well.
+
+I never stopped coding, though. I did small projects here and there both inside and outside of work, but I never
+really had the time to do anything big. After a year of being out of engineering I started to get an itch to build
+and do technical work. I even had a few dreams where I was coding! I have dreams about all sorts of weird things
+and don't dwell on them, but something about my surreal dream edition of Visual Studio Code stuck with me.
```

Comment:
💯

Reply:
Need to hear more about this oneiric VSC!!